simtools/layout/layout_array.py

```diff
@@ -396,14 +389,24 @@ def _get_corsika_sphere_center(self, tel_name):
         Returns
         -------
         astropy.Quantity
-            Telescope sphere center value (0.0*u.m if sphere center is not defined).
+            Telescope sphere center value
```

Comment:
Why did you remove the `(0.0*u.m if sphere center is not defined)` if it's still true?

Reply:
I found it too detailed, but looking at it again I can see as well reasons to keep it (e.g., this might be important when reading the sphinx-generated documentation). So I add it again.